docker/Dockerfile

```diff
@@ -1,8 +1,8 @@
-ARG PHP_VERSION_SELECTED
+ARG PHP_VERSION_SELECTED=8.2
```

Comment:
It's to minimize the laxity in version determination when doing the GitHub action workflow!